main.go

```diff
@@ -138,6 +140,7 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	mgr.AddMetricsExtraHandler("/debug/pprof/", http.HandlerFunc(pprof.Index))
```

Comment:
We should do this like here: https://github.com/kube-logging/logging-operator/blob/8a8a3ec6bd7764653cc049397d253c53b373a2c7/main.go#L158

Reply:
Probably we should also add these: 
```
mgr.AddMetricsExtraHandler("/debug/pprof/", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/heap", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/mutex", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/goroutine", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/threadcreate", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/block", http.HandlerFunc(pprof.Index))
		mgr.AddMetricsExtraHandler("/debug/pprof/cmdline", http.HandlerFunc(pprof.Cmdline))
		mgr.AddMetricsExtraHandler("/debug/pprof/profile", http.HandlerFunc(pprof.Profile))
		mgr.AddMetricsExtraHandler("/debug/pprof/symbol", http.HandlerFunc(pprof.Symbol))
		mgr.AddMetricsExtraHandler("/debug/pprof/trace", http.HandlerFunc(pprof.Trace))
```